src/exports.jl

```diff
@@ -230,6 +230,7 @@ export
   # Macros
   @preserve_ortho,
   # Methods
+  AbstractMPS,
```

Comment:
Was this meant to get added to the exports of `ITensors`?

Reply:
Agreed, don't need this one.